syntax = "proto3";

package Database;

message MapII {
    int32 key = 1;
    int32 val = 2;
}

message Game {
    int32 game_id = 1;
    int32 platform = 2;
    string name = 3;
    string appid = 4;
    string secret = 5;
}

message Role {
    int64 role_id = 1;
    int32 game_id =2;
    string openid = 3;
    string name = 4;
    string avatarurl = 5;
    string country = 6;
    string city = 7;
    int32 create_time = 8;
    int32 login_time = 9;
    int32 logout_time = 10;
    int32 daily_reset_time = 11;
};

message TimerField {
    repeated MapII tfs = 1;
};